digiwf-engine/digiwf-engine-rest-service/src/main/resources/application.yml

```diff
@@ -46,11 +46,31 @@ spring:
     url: '${ENGINE_DATASOURCE_URL:jdbc:h2:file:~/backenddb;DB_CLOSE_ON_EXIT=FALSE}'
     username: '${ENGINE_DATASOURCE_USER:sa}'
     password: '${ENGINE_DATASOURCE_PASSWORD:}'
-    driver-class-name: '${ENGINE_DATASOURCE_DRIVERCLASSNAME:org.h2.Driver}'
+  ldap:
+    urls: '${LDAP_URL}'
+    username: '${LDAP_USERNAME}'
+    password: '${LDAP_PASSWORD}'
 
 digiwf:
   security:
     client-id: ${SSO_ENGINE_CLIENT_ID}
     client-secret: ${SSO_ENGINE_CLIENT_SECRET}
+  ldap:
+    group-base: '${LDAP_GROUP_BASE}'
+    user-base: '${LDAP_USER_BASE}'
 
+logbook:
+  predicate:
+    include:
+      - path: "/engine-rest/engine/default/authorization/**"
+      - path: "/engine-rest/engine/default/group/**"
+  filter.enabled: true
+  secure-filter.enabled: true
+  format.style: http
```

Comment:
Frage ist was man hier mit Logbook macht. Nachdem das schon recht praktisch war würde ich das einfach in die Logging Lib verschieben, wenn da nichts dagegen spricht.

Reply:
Ja, dachte ich mir auch, dass das eher dort hin gehört

Reply:
Dependency in Logging-Lib verschoben, Properties habe ich jetzt mal nicht übernommen, sollten bis auf die Filter so Standard sein